Gemfile.lock

```diff
@@ -38,8 +38,8 @@ GEM
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-    addressable (2.8.0)
```

Comment:
we shouldn't update all this dependencies if they are not needed for the review apps, maybe they can be in another PR